config/config.go

```diff
@@ -261,5 +265,7 @@ func InitConfigFromFile(path string) *Config {
 
 	c.Process()
 
+	ExtraHeadersToLog = c.ExtraHeadersToLog
```

Comment:
We are globally exposing a list of headers we want to log. It works but is a bit ugly.

This is because we don't have access to the config struct when calling MakeRecord() and we did not want to refactor the world without guidance.